### ISSUE TITLE
bug: listing all devices of bluetooth 

### ISSUE THREAD

**Issue by Chromicle** [opendatakit/skunkworks-crow#346]
#### Software and hardware versions 
Android Samsung J8

#### Problem description
Currently, we are showing all the list of Bluetooth devices within the range  as our app is used by only android devices so, no need to show other devices like laptop, headphones, etc so, restrict to show only android devices

#### Steps to reproduce the problem
Use the transferring method as Bluetooth
Click receive and scan devices 
observe the behavior that shows all type of devices

#### Expected behavior
Show  only smartphones which turned on Bluetooth within range

**Comment by Chromicle** [comment]
@opendatakit-bot claim

**Comment by lakshyagupta21** [comment]
> Show only smartphones which turned on Bluetooth within range

Is there any way to identify whether the device is smartphone, smartwatch, laptop or something else.

**Comment by Chromicle** [comment]
Yeah we can know that with help of `getBluetoothClass` please have a look on [this](https://stackoverflow.com/questions/5989847/find-type-of-bluetooth-device-whether-mobile-or-laptops-or-mac-pcs)
